### PR TITLE
Mac install feedback 2020

### PR DESCRIPTION
**Pull request by FSchierok**
Update MacOsX install instructions with feedback from 2020

### CODE REVIEW

File: install/macos.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ xcode-select --install
 ```
 Danach "Installieren" auswählen und warten. Der Download wiegt etwa 130MB.
 
+ ### Git einstellen:
```
Comment (chrbeckm) [inline install/macos.md:52]: remove space before first `#`

File: install/macos.md [hunk 2]
```diff
@@ -60,20 +69,20 @@ __Wichtig__: Wir wollen Python 3.8 Graphical Installer (unten auf der Seite).  D
...
 ```bash
 git config --global core.editor "codium --wait"
 ```
 
-- Damit VS Code LaTeX-Code besser darstellen kann, muss das Plugin `LaTeX language support`
+- Damit VS Code LaTeX-Code besser darstellen kann, muss das Plugin `LaTeX Workshop`
   installiert werden
     - Mit `Shift+cmd+x` die Extensions öffnen
-    - `LaTeX language support` suchen
+    - `LaTeX Workshop` suchen
```
Comment (chrbeckm) [inline install/macos.md:83]: add instructions from the blogpost from Ludwig
https://toolbox.pep-dortmund.org/blog/index.html

File: install/macos.md [hunk 3]
```diff
@@ -82,7 +91,7 @@ git config --global core.editor "codium --wait"
 
 Im terminal:
```
Comment (chrbeckm) [inline install/macos.md:92]: "T"erminal

File: install/macos.md [hunk 4]
```diff
@@ -190,7 +199,7 @@ Im Terminal:
 
 Im Terminal:
 
-    pip install -U uncertainties
+    pip3 install -U uncertainties
```
Comment (maxnoe) [inline install/macos.md:202]: Nein! Wenn man die insallation richtig macht, reicht pip

File: install/macos.md [hunk 5]
```diff
@@ -60,29 +69,35 @@ __Wichtig__: Wir wollen Python 3.8 Graphical Installer (unten auf der Seite).  D
...
 ### Uncertainties
 
 - [Uncertainties](http://pythonhosted.org/uncertainties/): Python-Bibliothek
   für automatisierte Fehlerrechnung
 
-Im terminal:
+Im Terminal:
 
-    pip install uncertainties
+    pip3 install uncertainties
```
Comment (maxnoe) [inline install/macos.md:100]: pip

File: install/macos.md (hunk 2 above)
Comment (maxnoe) [reply install/macos.md:83]: language support is available now in codium, we should switch back.

File: install/macos.md (hunk 5 above)
Comment (maxnoe) [inline install/macos.md:100]: no. The correct form is either `python3 -m pip` which is the general best practive, or just `pip` when relying on that anaconda is first in path. `pip3` doesn't make sense.

File: install/macos.md [hunk 6]
```diff
@@ -60,27 +69,33 @@ __Wichtig__: Wir wollen Python 3.8 Graphical Installer (unten auf der Seite).  D
...
+- Damit VS Code LaTeX-Code besser darstellen kann, muss das Plugin `LaTeX Workshop`
+  installiert werden:
     - Mit `Shift+cmd+x` die Extensions öffnen
-    - `LaTeX language support` suchen
+    - `LaTeX Workshop` suchen
     - `install` anklicken
-    - VS Code neustarten
+    - VS Codium neustarten
+
+- Um LaTex Workshop für für uns richtig einzustellen:
```
Comment (maxnoe) [inline install/macos.md:87]: LaTeX

File: install/macos.md [hunk 7]
```diff
@@ -151,7 +154,7 @@ Im Terminal:
 
 Es sollte folgende Ausgabe erscheinen:
 
-    This is LuaTeX, Version 1.0.7 (TeX Live 2018)
+    This is LuaTeX, Version 1.X.X (TeX Live 2020)
```
Comment (maxnoe) [inline install/macos.md:157]: `1.13.2 (TeX Live 2021)`

**Comment by chrbeckm** [comment]
Does this pr needs to be merged, as it seems to have worked without the changes last years?

**Comment by jpherdi** [comment]
Dieser PR scheint veraltet zu sein. Die Infos hier sind bereits aktueller auf der Website.

**Pull request closed without merging**